src/codeu/chat/UserMap.java

```diff
@@ -0,0 +1,29 @@
+package codeu.chat;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+
+import codeu.chat.client.simplegui.NewAccount;
+
+/**
+ * Created by Suveena on 5/24/17.
+ */
+
+/* Basic structure that holds all the users. Create this map upon starting GUI.
```

Comment:
use /** (2 *s) for javadoc

Also prefer to align similar to suveena's comment above